### ISSUE TITLE
Webapi Swager schema generation fail in case when Get endpoint has param with Extension Attributes

### ISSUE THREAD

**Issue by swnsma** [magento/magento2#24116]
### Preconditions (*)
1. Magento 2.3.1 & 2.3-develop
2. Created and enabled custom module 'DreamVendor\DreamModule'.

### Steps to reproduce (*)
1. Create php interface which will be used for the endpoint. E.g.
```
namespace DreamVendor\DreamModule\Api;

use Magento\Framework\Api\Search\SearchResultInterface;

/**
 * My Custom Endpoint API.
 */
interface MyCustomEndpointInterface
{
    /**
     * Get some awesome stuff!
     *
     * @param \DreamVendor\DreamModule\Api\Data\SearchRequestInterface $searchRequest
     *
     * @return \Magento\Framework\Api\Search\SearchResultInterface
     */
    public function execute(\DreamVendor\DreamModule\Api\Data\SearchRequestInterface $searchRequest): SearchResultInterface;
}
```
2. Create SearchRequestInterface. Make it extensible with extension attributes.
```
namespace DreamVendor\DreamModule\Api\Data;

use Magento\Framework\Api\ExtensibleDataInterface;

/**
 * Search Request Data Object.
 */
interface SearchRequestInterface extends ExtensibleDataInterface
{
    /**
     * Get Extension Attributes.
     *
     * @return \DreamVendor\DreamModule\Api\Data\SearchRequestExtensionInterface|null
     */
    public function getExtensionAttributes(): ?\DreamVendor\DreamModule\Api\Data\SearchRequestExtensionInterface;

    /**
     * Set Extension Attributes.
     *
     * @param \DreamVendor\DreamModule\Api\Data\SearchRequestExtensionInterface|null $extension
     */
    public function setExtensionAttributes(?\DreamVendor\DreamModule\Api\Data\SearchRequestExtensionInterface $extension): void;

```
3. Declare your endpoint via `webapi.xml` in `etc` folder of your module.
```
<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Webapi:etc/webapi.xsd">
    <route url="/V1/my-custom-endpoint" method="GET">
        <service class="DreamVendor\DreamModule\Api\MyCustomEndpointInterface" method="execute"/>
        <resources>
            <resource ref="anonymous" />
        </resources>
    </route>
</routes>

```
4. Flush caches.
5. Try to generate schema. Go to `your.website.com/swagger/`.

### Expected result (*)
1. The page has been opened.
2. You can see your new endpoint.

### Actual result (*)
1. You can see message like `Failed to load API definition. Internal Server Error http://magento2.local/rest/all/schema?services=all`
2. Error message Notice: Undefined index: parameters in Magento/Webapi/Model/Rest/Swagger/Generator.php on line 741

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Charlie 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-19179` were created

**Issue Available**: @engcom-Charlie, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @swnsma. Thank you for your report. 
The issue has been fixed in magento/magento2#24117 by @swnsma in 2.3-develop branch
Related commit(s):
 - [166a1e0cf36a8e511b8be6aadec50fec81782fc8](https://github.com/magento/magento2/commit/166a1e0cf36a8e511b8be6aadec50fec81782fc8)

The fix will be available with the upcoming 2.3.4 release.
